modules/mobile/spec/request/appointments_list_request_spec.rb

```diff
@@ -39,7 +39,18 @@
         end
       end
 
-      it 'defaults to a range of -1 year and +1 year' do
+      it 'date range defaults to the beginning of the previous year to one year from now' do
+        expect_any_instance_of(Mobile::V0::Appointments::Service).to receive(:fetch_va_appointments).with(
```

Comment:
I think we could get more out of these tests if we combine this context and the `with an invalid date in params` context below into a single describe block. 
```
describe 'start and end date' do
  context 'when omitted from query params'
    it 'defaults to beginning of the previous year to one year from now'
    
  context 'when provided start date is after beginning of the previous year'
    it 'defaults to the beginning of the the previous year'
    
  context 'when provided start date is before the beginning of the previous year'
    it 'uses the provided start date'
    
  context 'when provided end date is before one year from now'
    it 'defaults to one year from now'
    
  context 'when provided end date is after one year from now'
    it 'uses the provided end date'
  
  context 'when provided dates are invalid'
    it 'returns a bad request error'
   
end
```

Reply:
You're right. This is much cleaner. I've made this change.